spec/rubocop/cop/naming/constant_name_spec.rb

```diff
@@ -25,6 +25,14 @@
     RUBY
   end
 
+  it 'registers 1 offense if rhs is offending const assignment' do
+    inspect_source(<<-RUBY.strip_indent)
```

Comment:
`expect_offenses` did not work with this example, so I adopted `inspect_source` to demonstrate the offense.